### PR TITLE
replace outdated dconf notice - fix issue 16040

### PR DESCRIPTION
**Pull request by wilzbach**
It will probably take a couple of weeks until the videos are ready, so this was the best that came to my mind as filler.

### CODE REVIEW

File: index.dd [hunk 1]
```diff
@@ -83,11 +83,9 @@ $(DIVID news,
 $(DIVC boxes,
     $(DIVC row,
         $(TOUR newspaper-o, News,
-            $(P $(LINK2 http://dconf.org/2016, DConf 2016) is coming up: May 4-6
-                in Berlin, Germany.
-                $(LINK2 http://dconf.org/2016/registration.html,
-                    Secure your seat)
-                before it's sold out!
+            $(P The D foundation got accepted for the Google Summer of Code 2016
+                and $(LINK https://summerofcode.withgoogle.com/organizations/5078256051027968/, four projects)
```
Comment (aG0aep6G) [inline index.dd]: Should be `LINK2`. But this has already been merged, so: #1305. @schuetzm, please don't pull so super quickly.

Comment (wilzbach) [inline index.dd]: Sorry about the typo :/

Comment (schuetzm) [inline index.dd]: @aG0aep6G Sorry, this looked so harmless...

Comment (wilzbach) [inline index.dd]: @schuetzm I am really sorry about that. I did this on the run :/
Those d macros are still tricky to me :/

Comment (CyberShadow) [inline index.dd]: Good thing we don't deploy on merge ... yet ... :)

**Comment by wilzbach** [comment]
> Good thing we don't deploy on merge ... yet ... :)

Would still be a good thing to have in the future as the conference announcement is still there.

**Comment by wilzbach** [comment]
> Would still be a good thing to have in the future as the conference announcement is still there.

I opened a [bug report](https://issues.dlang.org/show_bug.cgi?id=16076), but in general this seems rather easy

1) download dmd nightly with @MartinNowak's build script
2) build entire web page
3) upload to the dlang s3 bucket

```
curl -fsS https://dlang.org/install.sh | bash -s dmd-beta
make -f posix.mak
// pip install --user aws (if not available)
aws s3 cp --acl public-read <folder> s3://<bucket>
```

Most CI (Travis, CircleCI, ...) provider allow to define secure enviroment variables within the web interface.

**Comment by CyberShadow** [comment]
We already have http://wiki.dlang.org/User:Vladimir_Panteleev/Website_staging so it's not a matter of implementation, just everyone agreeing that it's what we want to do

**Pull request merged**